meetings/tool/src/main/resources/create-meeting_eu.properties

```diff
@@ -16,6 +16,7 @@ no_notification= Bat ere ez - Jakinarazpenik gabe
 notifications= Jakinarazpenak
 open_date= Hasteko data
 participants_selection= Partaideen hautaketa
+enable_coorganizers= Gehitu guneko irakasleak ko-antolatzaile gisa
```

Comment:
@pablosanroman please can you check? Thanks!

Reply:
hi   @bgarciaentornos! here it is: 
enable_coorganizers= Gehitu guneko irakasleak antolatzaile-kide gisa